src/utils/normalizeStyles.ts

```
@@ -15,6 +15,10 @@ const normalizeStyles: StyleProp<any> | StyleProp<any>[] = normalizeStylesMemo(
             return styles.map(styleObj => normalizeStyles(styleObj, currentTheme));
         }
 
+        if (!styles) {
+            return null;
```

Why return null?
Maybe return an empty object to make the return type consistent?

---

styles is required here, so, it should always exist. If not, should throw an error.

---

styles accepts an array, undefined, null or object.

---

It think the type should be a little better, then. 
StyleProp<any> isn't extactly great
It's a techdebt, I suppose